### DISCUSSION TITLE
videos and android app :)

### DISCUSSION THREAD

**Discussion by AlvMG** [bpatrik/pigallery2#461] [General]
Hi guys,
thanks for your hard work. To me this is by far the best web gallery that have all my requierements. I don´t know why the nightly build is not working for me but I´ve found the way of get the latest build working fine with the generation of the thumbnails, so all fine for me at the moment.
I have two suggestions. First of all, is it possible to add a configuration to convert only the videos that are on h265 codec? I have a lot of videos and I don´t want to encode tons of GB that are on 1080p, only want to convert the videos that I can´t see on the browser due to the compatibility of the codec.
Also, is there a posibility to create an android app to fix this issue on android devices?

Thanks,
Alvaro

**Comment by bpatrik** [comment]
Hi,

I'm happy that you like the project :)

Unfortunately there is no way now to filter on what video gets converted. 

I do not mind if you add a config flag that converts on a set op extensions.

Here is where it is decided if a video should be converted:

https://github.com/bpatrik/pigallery2/blob/baa38fe3559175e8ce094fa84777883e954000e8/src/backend/model/jobs/jobs/VideoConvertingJob.ts#L20-L23

Pigallery is just my hobby project that I tweak in my free time. Unfortunately I have no time and intention for creating an android app. 
But feel free to create one, the App communicates over standardish REST api.
